WallPaperDev/WallPaperDev/Extensions/UnsplashCellIdentifiable.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  UnsplashCellIdentifiable.swift
+//  WallPaperDev
+//
+//  Created by Jackson Ho on 3/19/20.
+//  Copyright © 2020 Stephen Ouyang. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+protocol UnsplashIdentifiable {
+  var urlString: String { get }
+}
+
+@available(iOS 13.0, *)
```

Comment:
What are your thoughts on making the min app version 13? 

https://developer.apple.com/support/app-store/
https://david-smith.org/iosversionstats/

Reply:
I would love if I can do that since the percentage of conversion is already > 70%. Also since I don’t have owner ship of this project I can’t release it to TestFlight as often as I would’ve like.

Reply:
Let's do it then! 

What do you mean you do not have ownership of the project you can not release to TestFlight as frequently?

Reply:
I don't know if possible, but since Stephen first created the project the bundle is under his account, and I couldn't upload it to TestFlight. So I don't know if it is possible to change owner ship or not.

Reply:
I see, can he grant your Apple ID account developer permissions for that specific app?

Reply:
I honestly don't know and he hasn't reply to my request for months now.